shared/chat/routes.js

```diff
@@ -30,31 +30,31 @@ const createChannel = {
   children: {},
 }
 
-const manageChannels = {
-  component: ManageChannels,
-  tags: makeLeafTags({hideStatusBar: isMobile, layerOnTop: !isMobile}),
-  children: {
-    editChannel,
-    createChannel,
-  },
-}
-
-const retentionWarning = {
-  component: RetentionWarning,
-  children: {},
-  tags: makeLeafTags({layerOnTop: !isMobile}),
-}
-
-const infoPanelChildren = {
+// These are routes that can be children of the inbox,
+// conversation, or the info panel. If you're adding a
+// component that will be routeable from the inbox or info
+// panel, add it here
+const chatChildren = {
   createChannel,
   editChannel,
-  manageChannels,
+  manageChannels: {
+    component: ManageChannels,
+    tags: makeLeafTags({hideStatusBar: isMobile, layerOnTop: !isMobile}),
+    children: {
+      createChannel,
+      editChannel,
+    },
+  },
```

Comment:
maybe unrelated but really leave team doens't have header hoc, we should add it here